### ISSUE TITLE
TypeError when using mkdocs-1.0

### ISSUE THREAD

**Issue by jakubmisek** [lukasgeiter/mkdocs-awesome-pages-plugin#5]
with recent update of mkdocs, the plugin crashes if there is no `pages` config

```
File "c:\hostedtoolcache\windows\python\3.6.4\x64\lib\site-packages\mkdocs_awesome_pages_plugin\plugin.py", line 24, in on_config
     pages = Factory(options).create(config['pages'])
File "c:\hostedtoolcache\windows\python\3.6.4\x64\lib\site-packages\mkdocs_awesome_pages_plugin\factory.py", line 34, in create
     children = self._create_pages(config)
File "c:\hostedtoolcache\windows\python\3.6.4\x64\lib\site-packages\mkdocs_awesome_pages_plugin\factory.py", line 64, in _create_pages
     return [self.create_page(c) for c in config]

TypeError: 'NoneType' object is not iterable
```

**Comment by lukasgeiter** [comment]
Thanks for reporting this issue. I've now added support for MkDocs 1.0 and released v1.2.1

**Comment by jakubmisek** [comment]
Amazing, thanks!

**Comment by lukasgeiter** [comment]
Unfortunately even after my fix the plugin doesn't work properly with MkDocs 1.0 yet. I'm working on fixing it though. Please check out #7 if you'd like to follow the progress.

**Comment by lukasgeiter** [comment]
I'm happy to inform you that MkDocs 1.0 support has just landed in v2.0.0 🙂
